app/konnect/servicehub/service-documentation.md

```diff
@@ -3,108 +3,94 @@ title: Dev Portal Service Documentation
 no_version: true
 ---
 
-A core function of the Dev Portal is publishing Service descriptions and API specs. Developers can use the spec and corresponding descriptions to access, consume, and register new applications against your Services.
-
-Through Service Hub, you can also publish any Service in your catalog and its
-documentation to the Dev Portal. Publishing services to the Dev Portal is the only way to expose your service to developers. Once the Service is published and available to developers, they can apply for access by [registering](/konnect/dev-portal/dev-reg/) a developer account. You can also [manage](/konnect/dev-portal/access-and-approval/manage-devs/) access to the Dev Portal from the {{site.konnect_saas}} interface.
+A core function of the Dev Portal is publishing service descriptions and API specs. Developers can use the spec and corresponding descriptions to access, consume, and register new applications against your services.
```

Comment:
```suggestion
A core function of the Dev Portal is publishing service descriptions and API specs. Developers can use the Dev Portal to access, consume, and register new applications against your services.
```
I thought the core function was to expose services and allow developers to manage their own credentials for their applications.

Reply:
Yep, definitely a copy-paste typo from ages ago.